doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -257,6 +257,17 @@ Zephyr's Mesh security toolbox implementation uses third-party crypto library AP
   * :kconfig:option:`CONFIG_BT_MESH_USES_TFM_PSA` - Enables use of the `Trusted Firmware M`_ PSA API based security toolbox (default option for platforms that support TF-M).
     Zephyr's Mesh operates with open key values, including storing them in the persistent memory.
 
+For Bluetooth Mesh provisioning, the following should be considered:
+
+* The Provisioner should choose the strongest available algorithm.
```

Comment:
This has been already implemented since Mesh 1.1. support. Customer cannot impact on this anyhow. No reason to mention this in documentation.